src/main/scripts/cbas

```diff
@@ -29,6 +29,9 @@ def get_jump_url(config):
     Return the jumphost url with HTTP scheme if we're in testing mode,
     HTTPS otherwise.
     """
+    if "://" in config.jump_host:
```

Comment:
Feel free to adapt the tests and remove this complexity :-) I just wanted to be able to use the production auth server with my test c-bastion instance.